src/tasks/create-first-git-commit.ts

```diff
@@ -18,11 +42,38 @@ export async function createFirstGitCommit(
           "lib/forge-std",
         ],
         {
-          cwd: path.resolve(targetDir, "packages", "foundry"),
+          cwd: foundryWorkSpacePath,
+        }
+      );
+      await execa(
+        "git",
+        [
+          "submodule",
+          "add",
+          "https://github.com/OpenZeppelin/openzeppelin-contracts",
+          "lib/openzeppelin-contracts",
+        ],
+        {
+          cwd: foundryWorkSpacePath,
         }
       );
       await execa("git", ["submodule", "update", "--init", "--recursive"], {
-        cwd: path.resolve(targetDir, "packages", "foundry"),
+        cwd: foundryWorkSpacePath,
+      });
+      await checkoutLatestTag(
+        path.resolve(foundryWorkSpacePath, "lib", "forge-std")
+      );
+      await checkoutLatestTag(
+        path.resolve(foundryWorkSpacePath, "lib", "openzeppelin-contracts")
+      );
+
+      // Commit the submodule changes in openzepplin-contracts
+      await execa("git", ["commit", "-am", "update forge-std"], {
```

Comment:
Had to commit add this extra commit in the submodule of openzepplin since there were some changes as compared to `master` and latest release `v4.9.3`